### DISCUSSION TITLE
any new webview exploit ?

### DISCUSSION THREAD

**Discussion by dudedecliedddddddd** [3kh0/ext-remover#1315] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I mostly use it at home and the only WebView I ever used was the sparkvue one but I accidentally deleted the app, the other web apps listed is blocked, and I cant install any apps from Google play, and chrome://chrome-signin is also blocked. You guys have anything else ?

**Comment by S-PScripts** [comment]
nuh uh
